Gemfile.lock

```diff
@@ -308,8 +309,5 @@ DEPENDENCIES
   unicorn (= 4.8.3)
   webmock (= 1.20.4)
 
-RUBY VERSION
```

Comment:
Try updating your version of bundler - it now states the ruby version in the lock file if you have one specified in the Gemfile.